core/metacling/src/TCling.cxx

```diff
@@ -1250,7 +1250,7 @@ static void RegisterCxxModules(cling::Interpreter &clingInterp)
 #elif defined(R__MACOSX)
          true
 #else // Windows
-         false
+         true
```

Comment:
We can probably drop the entire `supportedPlatform` variable altogether.

Reply:
right, thanks, I overlooked at this one ;-)

Reply:
Note that I don't understand what [this check](https://github.com/root-project/root/blob/master/core/metacling/src/TCling.cxx#L1266-L1269) is doing

Reply:
Is that still necessary? Eg, we basically enable the GMI everywhere, so maybe we can drop that check altogether.

Reply:
> Is that still necessary? Eg, we basically enable the GMI everywhere, so maybe we can drop that check altogether.

Removing [this code](https://github.com/root-project/root/blob/master/core/metacling/src/TCling.cxx#L1247-L1272)? You know better than me, I have no problem removing it all. Let me do it...

Reply:
That's a bit too more than I anticipated, however, this code was mostly for debugging purposes and we don't seem to have issues in that area anymore. So the advantage is that we drop another env variable that can modify ROOT's behavior. We can re-introduce this when necessary...

So yes, let's remove it.

Reply:
OK, so it should be good to merge now